src/utilities/idf/WorkspaceObject.cpp

```diff
@@ -211,7 +212,12 @@ namespace detail {
       }
       else {
         // return target's name
-        return oTarget->name();
+        if (auto s = oTarget->name()) {
+          return s.get();
+        } else {
+          // Fall back to handle if it doesn't have a name
+          return openstudio::toString(oTarget->handle());
+        }
```

Comment:
Second workaround I had to do that I didn't anticipate. IdfExtensibleGroup::fields() would call this one and crash because it doesn't have a name.